instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -304,6 +326,14 @@ def collect_custom_response_headers_attributes(response_headers):
     return attributes
 
 
+def _parse_status_code(resp_status):
+    status_code, _ = resp_status.split(" ", 1)
```

Comment:
Could we use some of this logic for `add_response_attributes` for span as well?

Reply:
That logic looks bit different because it sets error status with status string even when it's invalid.